sections/semantics-textlevel.include

```diff
@@ -2993,22 +2993,41 @@
       This example shows how the <{bdo}> element can be used to alter directionality formatting:
 
       <xmp highlight="html">
-        <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+        <bdo dir="rtl">English is a left-to-right language.</bdo>
       </xmp>
 
-      <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+      <bdo dir="rtl">English is a left-to-right language.</bdo>
 
-      This effect is achieved by applying the <code>rtl</code> attribute to the <{bdo}> element.
+      This effect is achieved by applying the <code>dir</code> attribute (with a value of 'rtl' indicating a reading from right to left direction) to the text inside the <{bdo}> element.
+  </div>
+
+  <p class="note">
+    The effect of using the <{bdo}> element is to <em>completely disable</em> the bidirectional algorithm. This means that characters within the <{bdo}> element will be displayed exactly as stored in memory and in the direction specified.
+    This is different than the <{bdi}> element, where the characters will be re-ordered and re-arranged according to the directional algorithm. In most cases, you will probably want to use <{bdi}> element. Use the <{bdo}> element only if you specifically want to disable the bidirectional algorithm completely for that text.
+  </p>
+
+  <div class="example">
+  The following example shows the difference in rendering between the <{bdi}> and <{bdo}> element.
+    <xmp highlight="html">
+    <bdi dir="rtl">שלום Majima! מה שלומך היום?</bdi>
+    <bdo dir="rtl">שלום Majima! מה שלומך היום?</bdo>
+    </xmp>
+    <bdi dir="rtl">שלום Majima! מה שלומך היום?</bdi><br>
+    <bdo dir="rtl">שלום Majima! מה שלומך היום?</bdo>
+
+    In this example, you can see the latin characters in the <{bdi}> element being rendered from left-to-right, whereas the rest of the text being rendered right-to-left. This is because the bidirectional algorithm recognizes latin characters and automatically applies a left-to-right rendering for just those characters.
+
+    In the <{bdo}> text, the bidirectional algorithm is disabled. So the text will be rendered character by character in the direction specified, with no correction even for the latin characters.
```

Comment:
Suggest: "with no correction even for the latin characters" -> "in a sequential order corresponding to that of the characters in memory"